build.gradle

```diff
@@ -50,12 +51,19 @@ ext {
     robolectricVersion = '4.3.1'
     protocVersion = '3.14.0'
     javaliteVersion = '3.14.0'
+
+    gMavenRoot = 'https://dl.google.com/dl/android/maven2'
+    firebaseDefaultPreguardFile='oss/default-preguard.txt'
```

Comment:
does this path exist? I can't see the oss directory in the repo

Reply:
Removed